QUEST_LV_0100.tsv

```diff
@@ -5769,11 +5769,11 @@ QUEST_LV_0100_20150317_005768	$Go to the Altar of Nugriautas Sanctum
 QUEST_LV_0100_20150317_005769	{memo X}
 QUEST_LV_0100_20150317_005770	Use the seal of space
 QUEST_LV_0100_20150317_005771	{memo X}
-QUEST_LV_0100_20150317_005772	Hidden mausoleum book appeared
-QUEST_LV_0100_20150317_005773	A book hidden somewhere in the 5th floor of mausoleum appeared. Talk to it. 
+QUEST_LV_0100_20150317_005772	Hidden Mausoleum Book appeared
+QUEST_LV_0100_20150317_005773	A book hidden somewhere in the 5th floor of Mausoleum appeared. Talk to it. 
```

Comment:
A book hidden somewhere in the 5th floor of the Royal Mausoleum appeared. Talk to it.

**or**

A book hidden somewhere in Royal Mausoleum 5F appeared. Talk to it.